presto-main/src/main/resources/webapp/assets/worker.js

```diff
@@ -78,6 +78,11 @@ let WorkerStatus = React.createClass({
         new Clipboard('.copy-button');
     },
     renderPoolBar: function(name, pool) {
+        if (pool === undefined)
+        {
```

Comment:
This doesn't seem to match the code style of the rest of this file.  @raghavsethi can you take a look at this?